code/modules/security_levels/security_levels.dm

```diff
@@ -77,22 +77,15 @@ GLOBAL_DATUM_INIT(security_announcement_down, /datum/announcement/priority/secur
 						FA.overlays += image('icons/obj/monitors.dmi', "overlay_red")
 
 			if(SEC_LEVEL_GAMMA)
-				GLOB.security_announcement_up.Announce("Central Command has ordered the Gamma security level on the station. Security is to have weapons equipped at all times, and all civilians are to immediately seek their nearest head for transportation to a secure location. The station's Gamma armory has been unlocked and is ready for use.","Attention! Gamma security level activated!", new_sound = sound('sound/effects/new_siren.ogg'))
+				GLOB.security_announcement_up.Announce("Central Command has ordered the Gamma security level on the station. Security is to have weapons equipped at all times, and all civilians are to immediately seek their nearest head for transportation to a secure location.","Attention! Gamma security level activated!", new_sound = sound('sound/effects/new_siren.ogg'))
 				GLOB.security_level = SEC_LEVEL_GAMMA
 
-				move_gamma_ship()
-
 				if(GLOB.security_level < SEC_LEVEL_RED)
```

Comment:
What is this code even... this never gets triggered due to line 81. Unless I'm missing something.
Don't have to change this btw. Just a remark

Reply:
Yeah, this is the bit that used to call the shuttle. Line 81 just sets the security level on the global var, as opposed to the 'level' var used in this file.

Reply:
Nah I meant it is set to `SEC_LEVEL_GAMMA` on line 81 which is defined as `3`. Then here it checks if the level is below `SEC_LEVEL_RED` which is defined as 2.... so 3 < 2 is always false

Reply:
good point, and adding to that I don't even think we use var/obj/machinery/door/airlock/highsecurity/red on any of the maps